RELEASE/scripts/autoscend/paths/avatar_of_shadows_over_loathing.ash

```diff
@@ -0,0 +1,590 @@
+
+			//Check if there are already skill points
+			matcher my_skillPoints = create_matcher("You have <b>(\\d+)<\\/b> skill", page);
+			if(my_skillPoints.find())
+			{
+				int skillPoints = to_int(my_skillPoints.group(1));
+				auto_log_info("Skill points found: " + skillPoints);
+				while(skillPoints > 0)
+				{
+					if(!have_skill($skill[Punt])) //Banish for the day
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=21", true);
+					}
+					if(!have_skill($skill[Second Wind])) //Restore 50% max HP during combat
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=16", true);
+					}
+					if(!have_skill($skill[Free-For-All])) //Free kill
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=19", true);
+					}
+					if(!have_skill($skill[Hot Foot])) //Deal Mys in Fire Dmg and set enemy on fire
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=15", true);
+					}
+					if(!have_skill($skill[Competitive Instincts])) //+100% Meat
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=10", true);
+					}
+					if(!have_skill($skill[Noogie])) //Weaken and stun enemy
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=14", true);
+					}
+					if(!have_skill($skill[Fancy Footwork])) //25% Item Drops
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=6", true);
+					}
+					if(!have_skill($skill[Cheerlead])) //Cheerled (10 advs, +50% all stats)
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=18", true);
+					}
+					if(!have_skill($skill[Strong Back])) //Passive Mus +20
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=3", true);
+					}
+					if(!have_skill($skill[Stop Hitting Yourself])) //Deal Moxie in phys dmg and stun
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=17", true);
+					}
+					if(!have_skill($skill[Ripped Triceps])) //Damaging skills deal 25% more dmg
+					{
+						page = visit_url("choice.php?pwd&option=1&whichchoice=1495&use=points&whichsk=8", true);
+					}
+					skillPoints -= 1;
+				}
+			}
+			//If there are no skill points, we still want to buy skills outright
+			if(!have_skill($skill[Ribald Memories]) && my_level()>=1 && my_meat()>100 + meatReserve()) //passive sleaze res and sleaze dmg
+			{
+				page = visit_url("choice.php?pwd&whichchoice=1495&option=1&whichsk=1", true);
+				set_property("auto_aosolLastSkill", 1);
+			}
+			if(!have_skill($skill[Blasted Glutes]) && my_level()>=2 && my_meat()>200 + meatReserve()) //max hp +50%
+			{
+				page = visit_url("choice.php?pwd&whichchoice=1495&option=1&whichsk=2", true);
+				set_property("auto_aosolLastSkill", 1);
```

Comment:
Set prop to 2

Reply:
Hmm never mind, looks intentional. A downside of the current approach is no priority of skills beyond level available. I'm def not familiar enough with the path to suggest a better ordering. I'm game for leaving that to a future pr, if an update is even needed

Reply:
Previous comment was ordering for me at buying of skills. There is when using skill points, as you know

Reply:
The aosolLastSkill prop was added so that we didn't constantly visit the skill book.

For a better priority of skills, we'd need to rewrite this entire section's conditionals because 2 skills are unlocked at each level, compared to other avatar classes where all/most skills are unlocked and you get points to use each level. Since most skills are useful and relatively reasonably priced, I am not too concerned about the skill order.

Skill ordering with skill points could probably be better. I chose what I eye-balled to be the best skills but I'm not 100% set on what ones are as we get more skill points. For autoscend though, the first 4-5 of each class are very useful and probably should not be changed.